extensions/notebook/src/extension.ts

```diff
@@ -122,8 +123,8 @@ export async function activate(extensionContext: vscode.ExtensionContext): Promi
 	const untitledBookTreeViewProvider = new BookTreeViewProvider([], extensionContext, true, READONLY_BOOKS_VIEWID);
 	await untitledBookTreeViewProvider.initialized;
 
-	extensionContext.subscriptions.push(vscode.window.registerTreeDataProvider(BOOKS_VIEWID, bookTreeViewProvider));
-	extensionContext.subscriptions.push(vscode.window.registerTreeDataProvider(READONLY_BOOKS_VIEWID, untitledBookTreeViewProvider));
+	vscode.window.createTreeView(BOOKS_VIEWID, { treeDataProvider: bookTreeViewProvider });
```

Comment:
@chlafreniere moved these out in his PR - may want to merge that in before making changes here.